playbooks/roles/nginx/templates/edx/app/nginx/sites-available/health-check.j2

```diff
@@ -0,0 +1,31 @@
+# config to pass GCP load balancer health checks along to the
+# /heartbeat endpoint of the LMS and allow lighter weight checks
+# on /status/
+
+upstream health-check-backend {
+  {% for host in nginx_cms_gunicorn_hosts %}
```

Comment:
So `nginx_cms_gunicorn_hosts` forces CMS connection to be redirected to the LMS?

Reply:
Just a typo. Fixed.